### PR TITLE
ROC-4661 PDF failure action group and alert query

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
New secret not yet added

### CODE REVIEW

File: bulk-print-fail-alert.tf [hunk 1]
```diff
@@ -9,7 +9,7 @@ module "cmc-bulk-print-fail-alert" {
   frequency_in_minutes = 5
   time_window_in_minutes = 5
   severity_level = "3"
-  action_group_name = "${module.cmc-bulk-print-fail-action-group.action_group_name}"
+  action_group_name = "${module.cmc-bulk-print-fail-action-group.bpf_action_group_name}"
```
Comment (markry-kainos) [inline bulk-print-fail-alert.tf]: should this not be just `action_group_name`?

File: pdf-fail-alert.tf [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  app_insights_query = <<AIQ
+requests
+| join (exceptions | where customDimensions != "") on operation_Id
+| join (dependencies | where target == "cmc-pdf-service-prod.service.core-compute-prod.internal") on operation_Id
+| where name startswith "POST"
+AIQ
+  frequency_in_minutes = 5
+  time_window_in_minutes = 5
+  severity_level = "3"
+  action_group_name = "${module.cmc-pdf-fail-action-group.pdf_action_group_name}"
```
Comment (markry-kainos) [inline pdf-fail-alert.tf]: and here too: `action_group_name`

File: bulk-print-fail-alert.tf (hunk 1 above)
Comment (andrewwa-kainos) [reply bulk-print-fail-alert.tf]: it's updated to reflect the changed `output` from `bulk-print-fail-action-group.tf`, which was updated to distinguish it from the new PDF failure action group name

**Review by markry-kainos — APPROVED** [review]
👍

**Comment by andrewwa-kainos** [comment]
Awaiting creation of email distribution list and definition of secret in all envs before merging.

**Pull request merged**